dask_ml/wrappers.py

```diff
@@ -256,17 +259,16 @@ class Incremental(ParallelPostFit):
     >>> import sklearn.linear_model
     >>> X, y = make_classification(chunks=25)
     >>> est = sklearn.linear_model.SGDClassifier()
-    >>> clf = Incremental(est, classes=[0, 1])
-    >>> clf.fit(X, y)
+    >>> clf = Incremental(est)
+    >>> clf.fit(X, y, classes=[0, 1])
     """
     def __init__(self, estimator, **kwargs):
+        estimator.set_params(**kwargs)
```

Comment:
I think that calling `estimator.set_params` is the least surprising thing here. In general, I don't expect users to be passing additional kwargs here, just scikit-learn.